src/js/edu-benefits/1990/components/IntroductionPage.jsx

```diff
@@ -14,48 +14,37 @@ class IntroductionPage extends React.Component {
     return (
       <div className="schemaform-intro">
         <FormTitle title="Apply for education benefits"/>
-        <p>This application is equivalent to Form 22-1990 (Application for VA Education Benefits).</p>
-        <SaveInProgressIntro
-          messages={this.props.route.formConfig.savedFormMessages}
-          pageList={this.props.route.pageList}
-          resumeOnly
-          {...this.props.saveInProgressActions}
-          {...this.props.saveInProgress}/>
+        <p>Equal to VA Form 22-1990 (Application for VA Education Benefits).</p>
```

Comment:
Just want to be sure that the removal of the Save In Progress intro here was deliberate?

Reply:
@phiden. @jbalboni I didn't remove any SIP code. Jeff B took a look at the intro files earlier this week, so he would know best if the SIP code is correct as is.

Reply:
It's not, that should be added back. I thought it was there when I looked before, but could be wrong.